adapter/config/ingress/status.go

```diff
@@ -69,7 +71,7 @@ func NewStatusSyncer(mesh *proxyconfig.MeshConfig, client kubernetes.Interface,
 
 	var publishService string
 	if mesh.IngressService != "" {
-		publishService = fmt.Sprintf("%v/%v", options.Namespace, mesh.IngressService)
+		publishService = fmt.Sprintf("%v/%v", ingressNamespace, mesh.IngressService)
```

Comment:
ingress namespace and ingress service values are fixed-form to satisfy the ingress status syncer (a core k8s library)